jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/settings/MobileSettings.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.jdi.light.mobile.settings;
+
+import static com.epam.jdi.light.driver.get.DriverInfos.*;
+import static com.epam.jdi.light.mobile.driver.MobileDriverInfos.*;
+
+/**
+ * Created by Roman Iovlev on 20.03.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class MobileSettings {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MobileSettings' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)